modules/check_in/app/services/v2/chip/client.rb

```diff
@@ -156,6 +156,28 @@ def refresh_precheckin(token:)
         end
       end
 
+      ##
+      # HTTP DELETE call to the CHIP API to delete check-in/pre check-in data
+      #
+      # @param token [String] CHIP token to call the endpoint
+      #
+      # @return [Faraday::Response]
+      #
+      def delete(token:)
+        connection.delete("/#{base_path}/actions/deleteFromLorota/#{check_in_session.uuid}") do |req|
+          req.headers = default_headers.merge('Authorization' => "Bearer #{token}")
+        end
+      rescue => e
+        log_exception_to_sentry(e,
+                                {
+                                  original_body: e.original_body,
```

Comment:
Are there any PII concerns with logging the body of the response to sentry?

Reply:
No PII data passed from the downstream in case of any exception, @LindseySaari